### DISCUSSION TITLE
pricing plans

### DISCUSSION THREAD

**Discussion by Pshemas** [Budibase/budibase#14230] [Help]
Are there any plans to introduce a perpetual option (no subs) for self hosted Budibase?

Quite frankly there's zero chance the organization where I'd like to introduce Budibase would ever consider subscription for such tool. It's a non profit with quite tight budget. And in general - subs when you're self hosting are a bit on a dumb side.

**Comment by ConorWebb96** [comment]
Hey @Pshemas,

I'm not quite sure what you mean by perpetual? The free version is free forever.

Budibase do non-profit discounts. Feel free to reach out to a member of the [sales team](https://budibase.com/contact/) to discuss your use case.

**Reply by Pshemas** [reply]
by perpetual I mean without serious restrictions (as the current free one has) and paid once (no subscription base). You know, the thing that was standard not too long ago ;) , until companies like Adobe thought that milking the customers monthly is a good idea and suddenly total cost of all the software you use went haywire .
